flow/network.cpp

```diff
@@ -185,7 +185,7 @@ Future<Reference<IConnection>> INetworkConnections::connect(const std::string& h
 	Future<NetworkAddress> pickEndpoint =
 	    map(resolveTCPEndpoint(host, service), [=](std::vector<NetworkAddress> const& addresses) -> NetworkAddress {
 		    NetworkAddress addr = addresses[deterministicRandom()->randomInt(0, addresses.size())];
-		    addr.fromHostname = NetworkAddressFromHostname::True;
```

Comment:
Is this `NetworkAddressFromHostname::True` type still needed?

Reply:
Yes, see line 95.